### PR TITLE
Add dependabot section

### LINKED ISSUE #125: Add dependabot section
### Description of the feature

We should add a dependabot section to the How-To chapter of the docs. Many people are not familiar with its functioning and they could use some help on setting it up and understanding the way it works.

### Steps for implementing the feature

Write write and write! :)

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #125

### CODE REVIEW

File: doc/source/how-to/dependabot.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    updates:
+    - package-ecosystem: "pip" # See documentation for possible values
+        directory: "/requirements" # Location of package manifests
+        schedule:
+            interval: "daily"
+
+This file should be located in the ``.github`` folder of your repository for
+GitHub to detect it automatically. As it can be seen there are several main options:
+
+* **package-ecosystem**: which lets Dependabot know what is your package manager.
```
Comment (MaxJPRey) [inline doc/source/how-to/dependabot.rst:32]:
```suggestion
* **package-ecosystem**: which lets Dependabot know what your package manager is.
```

Comment (RobPasMue) [reply doc/source/how-to/dependabot.rst:32]: Thanks @MaxJPRey!

**Review by MaxJPRey — APPROVED** [review]

**Comment by MaxJPRey** [comment]
LGTM! Thanks for adding this section and mentioning the benefit of dependabot @RobPasMue .

**Comment by jorgepiloto** [comment]
Should we place dependabot within https://dev.docs.pyansys.com/how-to/packaging.html? What do you think, @RobPasMue @MaxJPRey?

**Comment by RobPasMue** [comment]
> Should we place dependabot within https://dev.docs.pyansys.com/how-to/packaging.html? What do you think, @RobPasMue @MaxJPRey?

Mmm I don't see the that it fits there adequately, though Packaging also provides information on requirements... I'd prefer to see it in a different section. But we can discuss about it. I can see your point on placing it there. @MaxJPRey what insight do you have?

File: doc/source/how-to/index.rst [hunk 2]
```diff
@@ -19,3 +19,4 @@ functionalities such as logging, data transfer, and app APIs.
    continuous-integration
    releasing
    repository-protection
+   dependabot
```
Comment (jorgepiloto) [inline doc/source/how-to/index.rst:22]: What about introducing "dependabot" before the releasing section?

**Comment by RobPasMue** [comment]
Just moved it to Packaging after talking to @MaxJPRey! =) No further move arounds needed.

**Review by jorgepiloto — APPROVED** [review]

**Comment by jorgepiloto** [comment]
Thanks for addressing this @RobPasMue. Feel free to merge when ready 👍🏽

**Comment by RobPasMue** [comment]
Merging!

**Pull request merged**